Models/models.js

```diff
@@ -30,3 +30,10 @@ exports.selectArticleById = (article_id) => {
     }
   });
 };
```

Comment:
Here you will need to use a checkIfArticleExists function to differentiate from an invalid article and a valid article with no comments. See notes for help: 

https://notes.northcoders.com/courses/js-back-end/advanced-error-handling